docs/01_cva6_user/Programmer_View.rst

```diff
@@ -55,9 +61,17 @@ RISC-V Privileges
    "S - Supervior"
    "U - User"
 
+*Applicability to configurations:*
 
-Note: The addition of the H Extension is in the process. After that, HS, VS, and VU modes will also be available.

```

Comment:
Idem. This chapter can not be reused for Design Documentation. In Design Documentation, the chapter body will contain something like "CVA6E6ZX supports Machine mode, Supervisor and User modes are not supported". In that case, this chapter need to be in a single file which will not be included in index.rst of Design Documentation.

Reply:
Give a look at the latest commit. With templating, this should address your concern.